pyigm/cgm/cgm.py

```diff
@@ -260,3 +260,94 @@ def write_json(self, outfil=None):
         # Finish
         print("Wrote {:s} system to {:s} file".format(self.name, outfil))
 
+    def stack_plot(self,to_plot,pvlim=None,maxtrans=3,return_fig=True,**kwargs):
```

Comment:
Is it not possible to link to the linetools method instead?
Or is there a good reason for this to be its own method here?

Reply:
Let's say I want to programmatically generate a bunch of stack plots for CGM systems (as in a CGM survey).  Each call to generate the plot would also have to include the logic to extract a list of AbsLines to include in the plot.  In a real situation, I might just want to plot O VI and H I systems for each CGM, but the survey spans a large range in redshift where O VI may be covered at all redshifts but Ly-alpha may not.  Or, the whole Lyman series may be covered but I don't want to include every single transition in the stack plot.  Lines 291-339 deal with such issues and save the user from duplicating a bunch of complicated code simply for the sake of using the linetools function.

Reply:
Sounds good.